### ISSUE TITLE
test_cmp_cli_data files?

### ISSUE THREAD

**Issue by paulidale** [openssl/openssl#13012]
Is anyone else seeing these files appear?

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	test/recipes/81-test_cmp_cli_data/Mock/test.cacerts.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.cert.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_cr.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_csr_ignored.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_days.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_days_far_future.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_days_zero.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_disable.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_implicit.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_issuer.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_kur.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_kur_minimal.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_newkey.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_newkeypass.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_newkeypass_file.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_newkeypass_ignored.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_newkeypass_no_prefix.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_oldcert.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_out_trusted.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_out_trusted_big.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_p10cr.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_popo.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_reqexts.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_revreason.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_sans_critical.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_sans_default.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_sans_dns.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_sans_dns_critical.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_sans_dns_ip.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_sans_nodefault.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_sans_two_dns.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_sans_two_ip.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.certout_sans_uri.pem
	test/recipes/81-test_cmp_cli_data/Mock/test.extracerts.pem
```

Should they be cleaned up by the test in question of added to `.gitignore`?

**Comment by mattcaswell** [comment]
Fix is in #12934

**Comment by nhorman** [comment]
PR merged, closing
